atlas/AtlasMl/atlasml/ml/pipeline_workflows.py

```
@@ -0,0 +1,353 @@
+import uuid
+import numpy as np
+import uuid
```

Ruff (0.12.2)

Redefinition of unused `uuid` from line 1

Remove definition: `uuid`

(F811)